### DISCUSSION TITLE
Images sometimes not loading in HighlightedPosts.astro area as static build served by nginx (lazyloading)

### DISCUSSION THREAD

**Discussion by gangajamuna** [arthelokyo/astrowind#82] [Q&A]
Hi,
I have a static build of the Astrowind theme deployed on a server with nginx. For some reason sometimes (not all the times) for example two of four images in the HighlightedPosts.astro area of the theme are not showing up. After one or maybe more browser refreshes the images are loading again. Everywhere else the images are working fine. Does anybody has an Idea how to fix this. A workaround would be fine also. For example how do I deactivate lazyloading for that area?

Thank you!

**Comment by gangajamuna** [comment]
Hey Guys,
I found a workaround myself.  In the GridItem.astro file instead of importing Picture from @astrojs/image/components I imported Image and used it instead of Picture. This seems to resolve the issue for me now...
For reference: [[](url)](https://github.com/withastro/astro/tree/main/packages/integrations/image/)
Thank you

**Comment by prototypa** [comment]
Thanks for the feedback @gangajamuna 

It has been difficult for me to understand why it happens, in fact I still do not understand it. I guess it's related to the structure the HTML of the HighlightedPosts widget had, different from the other `<Picture>` instances I've used.

I hope with the changes I made it has been resolved: https://github.com/onwidget/astrowind/commit/a8704cf6d578be2689ffec5cbd856995c4b7c819

Please confirm me. Thanks.

**Reply by gangajamuna** [reply]
Thank you @prototypa I will try it out tomorrow and let you know!!

**Reply by gangajamuna** [reply]
Hi @prototypa,
I just tried out your changes and they have resolved the Issue of the Pictures sometimes not loading. It has one small Drawback: Compared to the previous Html there is now a bigger Layout shift when reloading at that section. 
Thank you! And btw I love the astrowind template. Awesome job!

**Comment by glenlong** [comment]
Hey folks. I don't know if it is related but I found a similar behavior for the blog post list pages. It was definitely happening on this page:

astrowind/src/pages/[...blog]/[category]/[...page].astro

But also I think on the main blog page here:

astrowind/src/pages/[...blog]/[...page].astro

On the first load, some or all images wouldn't show, but then a shift-refresh would cause them to load.

P.S. I'm running locally in a dev environment.

**Reply by gangajamuna** [reply]
Hi @glenlong,
At the moment I don't use categories but in the other list pages I don't experience the issue.

**Comment by prototypa** [comment]
@glenlong and @gangajamuna 

Thanks to both. I will be checking in detail.
